Function/BookFunc/Helpers/RequestHelpers.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using Microsoft.Graph.Auth;
+
+namespace TeamsMeetingBookFunc.Helpers
+{
+    internal static class RequestHelpers
+    {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "securePassword used outside this scope")]
+        internal static T AddAuthenticationToRequest<T>(this T request, string username, string password) where T : IBaseRequest
+        {
+            var securePassword = new SecureString();
+            // you should fetch the password from Azure Keyvault
```

Comment:
We are already fetching the password from keyvault. This method knows nothing from where we are fetching the params. Comment is misleading.

Reply:
```suggestion

```